### PR TITLE
Add html with content

### PR DESCRIPTION
**Pull request by FaGru**

### CODE REVIEW

File: bookmarks.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    <link rel="stylesheet" href="./styles/index.css" />
+  </head>
+  <body>
+    <div class="app-grid">
+      <header><h1>Bookmarks</h1></header>
+      <main class="app-grid__scrollarea h-100vh">
+        <article>
+          <h2>Question</h2>
+          <p>Inhalt der Frage</p>
+          <p>Antwort der Frage</p>
```
Comment (mwarnck) [inline bookmarks.html]: I think the answer should appear under the button after clicking?

File: bookmarks.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+            <li>tag 4</li>
+          </ul>
+        </article>
+      </main>
+
+      <nav>
+        <a href="">Home</a>
+        <a href="">Bookmarks</a>
+        <a href="">Create Question</a>
+        <a href="">My Profile</a>
```
Comment (mwarnck) [inline bookmarks.html]: here you could already link to the other html pages in the brackets after the href attribute

File: create.html [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quiz App / create</title>
+    <link rel="stylesheet" href="./styles/index.css" />
+  </head>
+  <body>
+    <div class="app-grid">
+      <header><h1>Create a Question</h1></header>
+      <form action="" class="app-grid__scrollarea h-100vh" autocomplete="off">
```
Comment (mwarnck) [inline create.html:13]: I am not sure if it is possible to include a section inside a form but maybe I am wrong ;)

File: styles/colors.css [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+  background-color: blue;
+}
+
+.bg-red {
+  background-color: red;
+}
+
+.bg-green {
+  background-color: green;
+}
```
Comment (mwarnck) [inline styles/colors.css:11]: wasn´t the colors.css for your own build variables for colors which you can use in other css rules? beginning with the :root an in the brackets for example --color-dark-blue: #212d40.

**Review by mwarnck — COMMENTED** [review]
overall good structure and looks very nice. have a look at my few comment ;)

File: bookmarks.html (hunk 1 above)
Comment (FaGru) [reply bookmarks.html]: oh yes you are right 👍🏻

File: bookmarks.html (hunk 2 above)
Comment (FaGru) [reply bookmarks.html]: thanks, but i think i will do that tomorrow 😉

File: create.html (hunk 3 above)
Comment (FaGru) [reply create.html:13]: i am not sure too, but i will  try it

File: styles/colors.css (hunk 4 above)
Comment (FaGru) [reply styles/colors.css:11]: i placed the colors here in colors.css to try the "app-grid" in the preview window

**Pull request merged**